### DISCUSSION TITLE
change plots to another wallet and key

### DISCUSSION THREAD

**Discussion by jlopez-hub** [Chia-Network/chia-blockchain#8459] [Support]
Hello there

I have more than 200 k32 plots in two machines, in a wallet and a key, and I want to transfer the plots from one machine to another wallet and another key,

This is possible? and if so, how can it be done?

**Comment by loppefaaret** [comment]
you can move the drives to another machine, and then _import_ the key to the machine - but you cannot transfer the plots from one key, to another key! - just import the 24 mnemonic that was used on the machine with the plots

**Comment by BrandtH22** [comment]
@jlopez-hub , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
